packages/databricks-vscode/DATABRICKS.quickstart.md

```diff
@@ -10,95 +8,117 @@ The Databricks extension for VS Code allows you to develop for the Databricks La
 
 # Features
 
+-   Create a Databricks Project [Databricks Asset Bundle]()
```

Comment:
I think it should just be "Create Databricks Asset Bundles" (+ the right doc link)